src/pages/UpdateServers.js

```diff
@@ -240,8 +277,8 @@ class UpdateServers extends BaseUpdateWizardPage {
     this.setState({showReplaceModal: false, serverToReplace: undefined});
   }
 
-  handleDoneReplaceServer = (server, wipeDisk, installOS) => {
-    this.replaceServer(server, wipeDisk, installOS);
+  handleDoneReplaceServer = (server, theProps) => {
+    this.replaceServer(server, theProps);
```

Comment:
the old version was better because it was more explicit about what was really needed.

Reply:
the props will include extra params for UpdateServer to process...think I will keep all the extra stuffs in theProps especially when most of the process logic will be moved to UpdateServer..otherwise I need to pass wipe disk, installos, user/pass and selectedserverId...and some are not necessary there.